requirements/requirements.txt

```diff
@@ -33,6 +33,6 @@ pyOpenSSL >= 22.0.0
 pydateinfer==0.3.0
 pyarrow >= 10.0.1, < 10.1.0
 dataprep_ml
-grpcio-tools
 python-magic >= 0.4.27
+#python-magic-bin >= 0.4.14 #for M1 mac users, comment out the above line and uncomment this one
```

Comment:
Is this something that will affect _base MindsDB_ functionality? If not, let's move it to the `writer_handler` folder

Reply:
I'm not sure what's happening in my case, I do have the normal `python-magic==0.4.27` and `libmagic` installed through `brew` and it works on my M2 Pro 🤔 

I've reopened the issue so that we can get to the bottom of this. For now, remove the req from this PR and we can fix this properly in a separate PR later on.

Reply:
interesting.. I tried that but it didn't work for me for some reason :(. I will remove it for now then